xarray/tests/test_dask.py

```diff
@@ -182,6 +183,26 @@ def test_bivariate_ufunc(self):
         self.assertLazyAndAllClose(np.maximum(u, 0), xu.maximum(v, 0))
         self.assertLazyAndAllClose(np.maximum(u, 0), xu.maximum(0, v))
 
+    def test_compute_args(self):
+        a = DataArray([1, 2]).chunk()
+        expected = DataArray([1, 4])
+        b = a * a
+        # compute
+        b1 = b.compute(get=dask.multiprocessing.get)
+        assert b1._in_memory
+        assert_equal(b1, expected)
+        b2 = b.compute(get=dask.multiprocessing.get, num_workers=4)
+        assert b2._in_memory
+        assert_equal(b2, expected)
+        # load
+        b3 = b.load(get=dask.multiprocessing.get, num_workers=4)
+        assert b3._in_memory
+        assert_equal(b3, expected)
+        # persist
+        b4 = b.persist(get=dask.multiprocessing.get, num_workers=4)
+        assert b4._in_memory
+        assert_equal(b4, expected)
+
```

Comment:
Redundant with the test below?

Reply:
fixed, these should have been `Variable`s.